### DISCUSSION TITLE
Alternative image viewer rendering

### DISCUSSION THREAD

**Discussion by jacksongoode** [akurdunkar/gspt#18] [Ideas]
Could there be a possibility to allow image rendering with ASCII chars or terminal supported high resolution like this project might enable: https://github.com/hzeller/timg.

**Comment by akurdunkar** [comment]
This in fact is part of the [Roadmap](https://github.com/aditya-K2/gspt/discussions/16). The current TUI library does support rendering images with ASCII characters, and I am planning to use the same. `timg` looks like a solid option too. I will definitely have a look at it.
